conan/tools/gnu/autotools.py

```diff
@@ -63,11 +63,38 @@ def make(self, target=None):
         command = join_arguments([make_program, target, str_args, jobs])
         self._conanfile.run(command)
 
+    @staticmethod
+    def _collect_shared_lib_files(folder):
+        result = []
+        files = os.listdir(folder)
+        for f in files:
+            full_path = os.path.join(folder, f)
+            name, ext = os.path.splitext(f)
+            if ext == ".dylib" and not os.path.islink(full_path):
+                result.append(f)
+        return result
+
+    def _fix_macos_install_paths(self):
+        """
+        fix LC_ID_DYLIB in Macos
+        """
+        libdirs = getattr(self._conanfile.cpp.package, "libdirs")
+        for folder in libdirs:
+            full_folder = os.path.join(self._conanfile.package_folder, folder)
+            shared_libs = self._collect_shared_lib_files(full_folder)
+            for shared_lib in shared_libs:
+                command = "install_name_tool -id @rpath/{} {}".format(shared_lib,
```

Comment:
Is ``install_name_tool`` always installed? is it possible that this fail if it doesnt and there is some scenario that wouldn't require this?

Reply:
It's installed with the XCode command line tools with more tools like `otool`, so I think it should not be a problem